analytics/agma/model.go

```diff
@@ -0,0 +1,50 @@
+package agma
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
```

Comment:
We updated the OpenRTB library to version 20. Please merge with the master branch and change these imports to `"github.com/prebid/openrtb/v20/openrtb2"`